azure-pipelines.yml

```diff
@@ -0,0 +1,50 @@
+jobs:
+  - job: Build
+    displayName: 'Build'
+    pool: Cloud-Build
+    strategy:
+      matrix:
+        Release-Any:
+          Platform: 'Any CPU'
+          BuildConfiguration: 'Release'
+    workspace:
+      clean: outputs
+
+    steps:
+      - task: GitVersion@5
```

Comment:
Build version number differs from GUI based build. Maybe need to add a Gitversion.yml configuration file?

Reply:
Solved in: df5d64a